src/log/log_functions.c

```diff
@@ -1494,6 +1503,50 @@ static void flush_log_file(int i, bool periodic, bool close_fd)
 	file->buf_offset = 0;
 }
 
+static inline void rotate_log(int fd, log_file_t *file) {
+  struct stat st;
```

Comment:
Forwarded the ganesha coding guidelines mail to you.

I think your vimrc should like this while committing to this repo :
 set noexpandtab
set tabstop=8
set shiftwidth=8
set textwidth=80